src/main/java/org/opentripplanner/middleware/spark/Main.java

```diff
@@ -7,11 +7,8 @@
 import org.eclipse.jetty.http.HttpStatus;
 import org.opentripplanner.middleware.BasicOtpDispatcher;
 import org.opentripplanner.middleware.auth.Auth0Connection;
-import org.opentripplanner.middleware.controllers.api.AdminUserController;
-import org.opentripplanner.middleware.controllers.api.ApiUserController;
-import org.opentripplanner.middleware.controllers.api.TripHistoryController;
+import org.opentripplanner.middleware.controllers.api.*;
```

Comment:
I think there are a few places where wildcard imports have been substituted for single imports. Could you change this back? This setting needs to be modified in your IDE settings (I think you can change the threshold for switching to wildcard to 9999 or something like that).

Reply:
Updated IDE and removed wildcard imports